sale_by_packaging/models/sale_order_line.py

```diff
@@ -88,6 +90,9 @@ def _onchange_product_uom_qty(self):
         res = super()._onchange_product_uom_qty()
         if not res:
             res = self._check_qty_is_pack_multiple()
+        if "warning" in res.keys():
```

Comment:
Is it really a good test having "warning" to reset those fields ?

Reply:
Nope, definitely needs to be improved

Reply:
I changed it, also the test is still on the return value (warning) now it is only based on one function. Much clearer and safer.